CHANGELOG.md

```diff
@@ -74,6 +74,7 @@ Here is an overview of all new **experimental** features:
 
 ### Fixes
 
+- **General**: Fix CloudEventType to correct format ([#3533](https://github.com/kedacore/keda/issues/3533))
```

Comment:
Since this new feature ships in this release, we do not have to list it as a fix